File: webflux-learning/src/main/java/com/neuqsoft/webfluxlearning/WebFluxApplication.java
package com.neuqsoft.webfluxlearning;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.cloud.client.discovery.EnableDiscoveryClient;

/**
 * @author sunjiarui
 * @Date 2020/10/10
 */
@EnableDiscoveryClient
@SpringBootApplication
public class WebFluxApplication {
    public static void main(String[] args) {
        SpringApplication.run(WebFluxApplication.class, args);
    }
}

File: webflux-learning/src/main/java/com/neuqsoft/webfluxlearning/WebFluxApplication.kt
package com.neuqsoft.webfluxlearning

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.cloud.client.discovery.EnableDiscoveryClient

/**
 * @author sunjiarui
 * @Date 2020/10/10
 */
@EnableDiscoveryClient
@SpringBootApplication
class WebFluxApplication

fun main(args: Array<String>) {
    runApplication<WebFluxApplication>(*args)
}